src/components/comments/CreateComment.js

```diff
@@ -98,7 +98,11 @@ const CreateComment = ({createComment, parentId, postId, comment, username, crea
 const mapStateToProps = state => {
   return {
       selectedPost: state.main.selectedPost,
-      username: state.auth.username
+      
+    
+    
```

Comment:
Great stuff! Think this could be one line to be cleaner though. Just if you have a chance!